plugin/src/main/kotlin/me/gabber235/typewriter/entry/entries/EventEntry.kt

```diff
@@ -6,7 +6,11 @@ import me.gabber235.typewriter.entry.TriggerEntry
 import org.bukkit.entity.Player
 
 @Tags("event")
-interface EventEntry : TriggerEntry
+interface EventEntry : TriggerEntry {
+	// make a command val that does not need to be overridden
+	val command: String
+		get() = ""
```

Comment:
The event entry should not have arbitrary properties that most event's don't need. Instead make a separate interface for just commands like

```kotlin
interface CustomCommandEntry : EventEntry {
    val command: String
    // Maybe some abstract functions that a specific command entry can override
    // Things like `filter`, `onTrigger`
}
```

Allowing some additional methods to be ran can be helpfull, such as `filter` which lets you filter for the command, like only players can use this.
Or `onTrigger` which give implementors the ability to change what happens on trigger. Maybe they don't want to trigger all next entries but want to do something else.

This allows us later do define some other custom commands entries which may have different functionality.